docs/src/content/docs/es/guides/authoring-content.md

```diff
@@ -360,6 +360,31 @@ Un título opcional del bloque de código se puede establecer con un atributo `t
   ```
   ````
 
+## Details
```

Comment:
Just to double check, is it expected to have the same heading as the English version or should this be translated too like the other headings?

Reply:
Yeah, as far as I know it is. I went to MDN's docs for it. Here is the [example]( https://developer.mozilla.org/es/docs/Web/HTML/Element/details)